File: C++/abc006A.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
  int n,m;
  cin >> n;
  m = n % 3;
  if( m == 0){
    cout << "YES" << endl;
  }else{
    cout << "NO" << endl;
  }
}

File: JavaScript/abc006A.js
function Main(input) {
    const a = input.split('\n');
    const n = parseInt(a[0], 10);
    const m = n % 3;
    if(n == 0){
      console.log("YES");
    }else{
      console.log("NO");
    }
  }
  
  Main(require("fs").readFileSync("/dev/stdin", "utf8"));